### DISCUSSION TITLE
Upgrade wallet db to v2 when blockchain is already v2?

### DISCUSSION THREAD

**Discussion by cross** [Chia-Network/chia-blockchain#11024] [Support]
Instead of waiting the many hours that it was expected to take, I downloaded a recent v2 blockchain.  But, I cannot figure out how to upgrade my wallet db to v2.  Is that possible?  Is there an invocation of `chia db upgrade` that will do that?  Without arguments, it complains because I already have a v2 database.

**Comment by FL14** [comment]
all you have to do is del wallet DB
then start chia , wallet sync very soon
